src/app/header/header.component.ts

```diff
@@ -46,7 +49,21 @@ export class HeaderComponent implements OnInit {
   private subscription;
 
   constructor(private _authService: AuthService, private _router: Router, private _langService: LanguageService, private _badgeService: BadgeService,
-              private _notify: NotifyService, private _snackbar: MatSnackBar, private _leaderBoardService: LeaderBoardService, private _navBarService: NavbarService) {
+              private _notify: NotifyService, private _snackbar: MatSnackBar, private _leaderBoardService: LeaderBoardService, private _navBarService: NavbarService,
+              private updates: SwUpdate) {
+    updates.available.subscribe(event => {
+      swal({
```

Comment:
Can't you use the custom swal alert function in shared.data.service.ts here?

Reply:
@Raxerz Custom **swal** alert would not have buttons like _update_ and _cancel_.